src/browser/Color.ts

```diff
@@ -47,6 +47,17 @@ export namespace color {
     const rgba = channels.toRgba(r, g, b);
     return { css, rgba };
   }
+  export function blendSelectionWithBgWithOpacity(colors: IColorSet, opacity: number=0.3): void {
+    const [selectionR, selectionG, selectionB, selectionA] = rgba.toChannels(colors.selection.rgba);
+    // The selection color is opaque. It needs to be blended with background color at 0.3 opacity Issue #2737
+    if (selectionA === 0xFF) {
+      const newAlpha: number = Math.round(opacity*255);
```

Comment:
We tend to use hex when working with color channels:

```suggestion
      const newAlpha: number = Math.round(opacity * oxFF);
```